src/dialogs/more-info/ha-more-info-info.ts

```diff
@@ -45,9 +45,13 @@ export class MoreInfoInfo extends LitElement {
       <div class="container" data-domain=${domain}>
         ${!stateObj
           ? html`<ha-alert alert-type="warning">
-              ${this.hass.localize(
-                "ui.dialogs.entity_registry.editor.unavailable"
-              )}
+              ${this.entry?.disabled_by
+                ? this.hass.localize(
+                    "ui.dialogs.entity_registry.editor.entity_disabled"
```

Comment:
This is good, we should probably also add an action to the alert when disabled to enable the entity.

Reply:
Thought about it. But that is easily available on the config screen once you know it's disabled, and sort of mixes of roles of the screens.